docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -0,0 +1,5 @@
+= Truststore Improvements
+
+There is a new option truststore-paths that takes a list of paths that are either truststores, or directories containing truststores.  The files are expected to be in PEM format or pkcs12 (.p12 or .pfx) with no password required to access the certs.
+
+The merged truststore will be created from all the certs found along with the existing default truststore certs for the JRE.  The merged truststore will be set as the system default truststore and used as the default for the truststore accessed via the spi-truststore.
```

Comment:
`merged` is an implementation detail. What users care about is that they can list multiple certificates or truststores.